onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -218,7 +218,7 @@ TEST_F(CApiTest, custom_op_handler) {
   TestInference<PATH_TYPE>(env_, CUSTOM_OP_MODEL_URI, inputs, "Y", expected_dims_y, expected_values_y, 0, custom_op_domain);
 }
 
-#ifdef ENABLE_LANGUAGE_INTEROP_OPS
+#if defined(ENABLE_LANGUAGE_INTEROP_OPS) && !( defined(_WIN32) && defined(_DEBUG))
```

Comment:
Can you add a helpful comment here stating why this logic is required?

Reply:
Adding comments.